bayesian/utility/v2/ca_response_builder.py

```diff
@@ -129,29 +128,43 @@ def get_message(self):
             message += f"and {self.pvt_vul} security advisory with {len(self.severity)} " \
```

Comment:
for this message, why does it start with "and" . Will not the message be like "Django - 1.1.1 has and"

Reply:
@prashbnair Looks like you forgot to expand hidden lines.. :) 
Prior to line 129, on L128 `message += f"{self.public_vul} known security vulnerability"`